packages/account/src/Components/address-details/__tests__/address-details.spec.tsx

```diff
@@ -168,9 +191,9 @@ describe('<AddressDetails/>', () => {
     });
 
     it('should render AddressDetails component not svg', async () => {
-        mock_props.is_svg = false;
+        const new_props = { ...mock_props, is_svg: false };
```

Comment:
why do we need to create new object?

Reply:
I feel using immutable data is better as we know where the object has changed